### PR TITLE
Added UI for expense form

### PR DESCRIPTION
**Pull request by amansingh7010**

### CODE REVIEW

File: src/actions/expense.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {
+    SUBMIT_DAILY_EXPENSE,
+} from "../constants/expense";
+
+export const submitDailyExpenseForm = (data) => {
+    console.log('in action creator: ' + JSON.stringify(data));
```
Comment (manju4ever) [inline src/actions/expense.js]: remove console.log

File: src/components/DailyExpenseForm.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { reduxForm, Field } from 'redux-form';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { ExpenseActions } from '../actions';
+
+class DailyExpenseForm extends Component {
+
```
Comment (manju4ever) [inline src/components/DailyExpenseForm.js]: Please use a linting, use airbnb styling

File: src/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ import reducers from './reducers';
 const createStoreWithMiddleware = applyMiddleware()(createStore);
 
 ReactDOM.render(
-  <Provider store={createStoreWithMiddleware(reducers)}>
+  <Provider store={createStoreWithMiddleware(reducers,  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())}>
```
Comment (manju4ever) [inline src/index.js]: factor out store into a different module.

File: webpack.config.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     publicPath: '/',
     filename: 'bundle.js'
   },
+  devtool:'eval-source-map',
```
Comment (manju4ever) [inline webpack.config.js]: update build scripts and pass ENV variable for dev & prod builds.

File: src/reducers/index.js [hunk 5]
```diff
@@ -1,7 +1,10 @@
 import { combineReducers } from 'redux';
+import { reducer as formReducer } from 'redux-form';
+import expenseReducer from './expense';
 
 const rootReducer = combineReducers({
-  state: (state = {}) => state
+  form: formReducer,
+  expense: expenseReducer
```
Comment (chakravarthysm) [inline src/reducers/index.js]: missing ending comma

File: src/components/DailyExpenseForm.jsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+                        </div>
+                    </form>
+                </div>
+            </div>
+        );
+    }
+}
+
+const dailyExpenseForm = reduxForm({
+    form: 'dailyExpenseForm'
```
Comment (chakravarthysm) [inline src/components/DailyExpenseForm.jsx]: missing comma

File: src/components/DailyExpenseForm.jsx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+import { bindActionCreators } from 'redux';
+import { ExpenseActions } from '../actions';
+
+
+class DailyExpenseForm extends Component {
+
+    constructor() {
+        super();
+        this.state = {};
+        this.saveExpense  =this.saveExpense.bind(this);
```
Comment (chakravarthysm) [inline src/components/DailyExpenseForm.jsx]: lint?

**Pull request merged**